test/env_var_provider_test.exs

```diff
@@ -40,7 +40,8 @@ defmodule EnvVar.ProviderTest do
         server_count: %{type: :integer, default: "123"},
         name: %{type: :string, default: "grendel"},
         settings: %{type: {:list, :string}, default: "swarthy,hairy"},
-        keys: %{type: {:tuple, :float}, default: "1.1,2.3,3.4"}
+        keys: %{type: {:tuple, :float}, default: "1.1,2.3,3.4"},
+        empty_string_list: %{type: {:list, :string}, default: ""}
```

Comment:
Just checking ... and that means that I can configure an env_var in the deploy.yml file as `ENV_VAR: ""` and it will turn it into an empty list (of strings). Right?

What if we need an empty list of integers?

Reply:
@rolandtritsch, I added that